libs/api/domains/syslumenn/src/lib/syslumenn.resolver.ts

```diff
@@ -16,4 +17,11 @@ export class SyslumennResolver {
   getHomestays(@Args('input') input: GetHomestaysInput): Promise<Homestay[]> {
     return this.syslumennService.getHomestays(input.year)
   }
+
+  // Note: We need to be careful when caching the Auction data, as it's prone to changes only minutes before the auction takes place.
+  // TODO: Check if the CACHE_TIME in production. If it's only a few seconds, we should consider caching the data.
```

Comment:
Raising this TODO in a thread.

As stated in the code comment/todo, we need to be careful when caching the Auction data, as it's prone to changes only minutes before the auction takes place.

We need to find out what value `CACHE_TIME` is set to in production and after that assess how we'll handle caching of Auction data.

Reply:
We've decided not to add cache to this method, at least for now. cc. @Ingimarsson